scene/main/viewport.cpp

```diff
@@ -1338,8 +1338,8 @@ bool Viewport::_gui_call_input(Control *p_control, const Ref<InputEvent> &p_inpu
 				stopped = true;
 				break;
 			}
-			if (control->data.mouse_filter == Control::MOUSE_FILTER_STOP && is_mouse_event && !(is_scroll_event && control->data.force_pass_scroll_events)) {
-				// Mouse events are stopped by default with MOUSE_FILTER_STOP, unless we have a scroll event and force_pass_scroll_events set to true
+			if (control->data.mouse_filter == Control::MOUSE_FILTER_STOP && (is_mouse_event || p_input->get_device() == InputEvent::DEVICE_ID_TOUCH_MOUSE) && !(is_scroll_event && control->data.force_pass_scroll_events)) {
+				// Mouse events and emulated events from Mouse events are stopped by default with MOUSE_FILTER_STOP, unless we have a scroll event and force_pass_scroll_events set to true
```

Comment:
Can you update the logic to set `is_mouse_event` instead to include `p_input->get_device() == InputEvent::DEVICE_ID_TOUCH_MOUSE`.

That'll ensure that other uses of `is_mouse_event` also reflect the change.